components/apimgt/org.wso2.carbon.apimgt.persistence/src/test/java/org/wso2/carbon/apimgt/persistence/RegistryPersistenceImplTestCase.java

```diff
@@ -580,16 +584,21 @@ public void testAddAPI() throws RegistryException, APIPersistenceException, APIM
         GenericArtifactManager manager = Mockito.mock(GenericArtifactManager.class);
         PowerMockito.when(RegistryPersistenceUtil.getArtifactManager(registry, APIConstants.API_KEY))
                 .thenReturn(manager);
-
+        //add 1 year old timestamp since it is not the latest
+        LocalDateTime now = LocalDateTime.now().minusDays(365);
```

Comment:
Didn't we decide to make this 1 day or similar due to some reason?

Reply:
this is a test case, the code level change has done to find the calculated timestamp in https://github.com/wso2/carbon-apimgt/pull/10913/files#diff-19a032cb823fd4f31e79376db6427e305b79949a26979f42d5e7ea0c607cdff2R6961

Reply:
ack.